### PR TITLE
Add an exponential back off when watching pipelines

### PR DESCRIPTION
**Pull request by sbabitz**
Initial back off timer is 5 seconds, back off factor is 1.5 - up to 1 minute delay. 
Each event resets the back off timer to 5 seconds.

### CODE REVIEW

File: pipelines/internal/commands/watch/watch.go [hunk 1]
```diff
@@ -55,6 +55,9 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 
 func watch(ctx context.Context, service *genomics.Service, name string) (interface{}, error) {
 	var events []*genomics.Event
+	const minDelay = 5 * time.Second
```
Comment (kemp-google) [inline pipelines/internal/commands/watch/watch.go]: You never need 'minDelay' again... What about something like

delay := 5 * time.Second
...
time.Sleep(delay)
delay *= 1.5
...
if limit := time.Minute; delay > limit {
  delay = limit
}

Comment (sbabitz) [reply pipelines/internal/commands/watch/watch.go]: I do use minDelay once more - to reset the delay after each even

Comment (kemp-google) [reply pipelines/internal/commands/watch/watch.go]: OK so same basic changes but we can have const initialDelay and assign that originally

Comment (sbabitz) [reply pipelines/internal/commands/watch/watch.go]: ok I changed the name and removed the maxDelay from const (limit var in the if loop instead)

File: pipelines/internal/commands/watch/watch.go [hunk 2]
```diff
@@ -55,6 +55,8 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 
 func watch(ctx context.Context, service *genomics.Service, name string) (interface{}, error) {
 	var events []*genomics.Event
+	const initialDelay = 5 * time.Second
+	var delay = initialDelay
```
Comment (kemp-google) [inline pipelines/internal/commands/watch/watch.go]: delay := initialDelay (we typically use var only when an initial assignment isn't required)

**Comment by pgrosu** [comment]
@sbabitz What else do you think you are missing in the exponential backoff scheme here?

**Pull request merged**